src/aat/resources/features/F-129 - Access Control SC/SC_Public_Case_Created.td.json

```diff
@@ -0,0 +1,63 @@
+{
+	"_guid_": "SC_Public_Case_Created",
+	"_extends_": "Case_Creation_Data_Base",
+
+	"users": {
+    "invokingUser": {
+      "username": "auto.test.cnp@gmail.com",
+      "password": "Pa55word11"
+    }
```

Comment:
Should this not be : 

```
   "invokingUser": {
      "_extends_" : "CaseworkerAutotest"
   }
```

Reply:
ahh good catch, hardcoded when i was doing some debugging